repos/system_upgrade/common/actors/commonleappdracutmodules/files/dracut/90sys-upgrade/leapp_debug_tools.sh

```diff
@@ -0,0 +1,34 @@
+# library containing some useful functions for debugging in initramfs
+
+# mounts the sysroot
+leapp_dbg_mount() {
+    systemctl start sysroot.mount
```

Comment:
you'd probably want it to be remounted rw

`mount -o remount,rw /sysroot

Reply:
I was thinking about it. Okay I will make it rw